dali/operators/sequence/optical_flow/optical_flow.cc

```diff
@@ -19,38 +19,56 @@ namespace dali {
 
 
 DALI_SCHEMA(OpticalFlow)
-                .DocStr(R"code(Calculates the Optical Flow for sequence of images given as a input.
-Mandatory input for the operator is a sequence of frames.
-As an optional input, operator accepts external hints for OF calculation.
-The output format of this operator matches the output format of OF driver API.
-Dali uses Turing optical flow hardware implementation: https://developer.nvidia.com/opticalflow-sdk
+                .DocStr(R"code(Calculates the optical flow for a sequence of images that were
+provided as the input.
+
+The mandatory input for the operator is a sequence of frames. As an Optionally, the operator
+also accepts external hints for the optical flow calculation. The output format of this operator
+matches the output format of the optical flow driver API.
+Refer to https://developer.nvidia.com/opticalflow-sdk for more information about the
+Turing and Ampere optical flow hardware that is used by DALI. This operator allows sequence inputs.
 )code")
                 .NumInput(1, 2)
                 .NumOutput(1)
-                .AddOptionalArg(detail::kPresetArgName, R"code(Speed/quality level of OF calculation:
+                .AddOptionalArg(detail::kPresetArgName, R"code(Speed and quality level of the
+optical flow calculation.
 
-- 0.0 is the lowest speed and the best quality,
-- 0.5 is the medium speed and quality,
-- 1.0 is the fastest speed and the lowest quality.
+Here are the supported values:
```

Comment:
Not sure about this - are these the _only_ allowed values or just examples?

Reply:
I think the allowed values.

Reply:
Checked. When the values are outside the provided range it is assume that it is not defined.

Reply:
So, if it is a range it should be documented differently. Something like:
"Values must be in the range [0.0, 1.0], where 0.0 represents lowest speed and highest quality and 1.0 represents highest speed and lowest quality"